	Not needed, DISABLE_OUTLINE is no longer used in pdftex sources,
	but might be nevertheless useful.

	from:
		Han The Thanh <thanh@pdftex.org>
		Martin Schroeder <martin@pdftex.org>

diff -ur -N xpdf-3.02.orig/xpdf/PDFDoc.h xpdf-3.02/xpdf/PDFDoc.h
--- xpdf-3.02.orig/xpdf/PDFDoc.h	2007-02-27 23:05:52.000000000 +0100
+++ xpdf-3.02/xpdf/PDFDoc.h	2008-07-17 02:29:04.000000000 +0200
@@ -5,6 +5,9 @@
 // Copyright 1996-2003 Glyph & Cog, LLC
 //
 //========================================================================
+//  Modified for TeX Live by Peter Breitenlohner <tex-live@tug.org>
+//  See top-level ChangeLog for a list of all modifications
+//========================================================================
 
 #ifndef PDFDOC_H
 #define PDFDOC_H
@@ -171,13 +174,15 @@
   double pdfVersion;
   XRef *xref;
   Catalog *catalog;
+  GBool ok;
+  int errCode;
+/* Keep the size of the structure the same whether DISABLE_OUTLINE or
+   not, and put it last to try to find errors more quickly.  */
 #ifndef DISABLE_OUTLINE
   Outline *outline;
+#else
+  void *dummy;
 #endif
-
-
-  GBool ok;
-  int errCode;
 };
 
 #endif
